extra/resources/Dummy

```diff
@@ -221,6 +221,12 @@ dummy_monitor() {
             return $OCF_ERR_GENERIC
             ;;
 
+            40)
+            # monitor level that returns error code from state file
+            rc=$(cat ${OCF_RESKEY_state})
+            [ -n "$rc" ] && ocf_exit_reason "return code from ${OCF_RESKEY_state}" && return $rc
```

Comment:
You can come up with a better message than this ;) It's a dummy resource for testing, so we can have a little fun with it